storage/rocksdb/ha_rocksdb.cc

```diff
@@ -1261,14 +1285,6 @@ static MYSQL_SYSVAR_SIZE_T(compaction_readahead_size,
                           rocksdb_db_options->compaction_readahead_size,
                           /* min */ 0L, /* max */ SIZE_T_MAX, 0);
 
-static MYSQL_SYSVAR_BOOL(
```

Comment:
might need to deprecate these rather than straight remove, even if they are there without a purpose.